.eslintrc.js

```diff
@@ -28,123 +32,50 @@ module.exports = {
     node: true,
     jest: true,
   },
+  reportUnusedDisableDirectives: true,
```

Comment:
This config options makes eslint check for `eslint-disable` statements which are outdated and autoremoves them like in e.g. https://github.com/ferdium/ferdium-app/pull/1281/files#diff-7db082e9f1255575e774bf5f2c4b23b7ed970cedaf3b8a4cf583ac46d19f96b6L71